modules/vaos/app/services/vaos/v2/appointments_presentation_filter.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+# these filters were derived from the web app front end code base
+module VAOS
+  module V2
+    class AppointmentsPresentationFilter
+      def initialize
```

Comment:
Does web currently show requested/pending appts? Want to make sure we're not changing their behavior.

Reply:
So web should be using the `purposes[]` parameter, which will filter it to everything they want. If they have a view that's intended to show pending, it'll be included in the `purposes[]` parameter and they'll be shown. Otherwise, they'll be ignored